### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by jlcai5** [xiaolai-sqlai/mobilenetv3#22]
hi , my training loss is normal, but the validate loss is nan , Have you met anybody?

**Comment by wplf** [comment]
yes, i met this problem, too. 
Maybe it's the original learing rate is too big, and the model can't find the right weight space.
More epochs will solve this problem.
